Makefile

```diff
@@ -28,6 +28,7 @@ lint:  ## Lint go source code
 sync: sync-googleapis | testdata/conformance/google/protobuf ## Clone and copy conformance protos from GitHub
 	$(eval DEST := $(shell mktemp -d))
 	git clone --depth=1 https://github.com/protocolbuffers/protobuf.git $(DEST)
+	for patch in testdata/patches/*.patch; do patch -d $(DEST) -p1 < $${patch}; done
```

Comment:
your manual mac ci system confirms: it works (tm)

Reply:
:+1: